swarm/src/connection.rs

```diff
@@ -157,6 +161,8 @@ where
     local_supported_protocols: HashSet<StreamProtocol>,
     remote_supported_protocols: HashSet<StreamProtocol>,
     idle_timeout: Duration,
+    /// The counter of active streams
+    stream_counter: ActiveStreamCounter,
```

Comment:
This is unrelated, please remove this bit!

Reply:
Done. Please review it.